### DISCUSSION TITLE
RGBD Point cloud registration

### DISCUSSION THREAD

**Discussion by kopetri** [facebookresearch/pytorch3d#808] [Q&A]
## ❓ Questions on how to use PyTorch3D

Hello there,

I need some help on a camera pose problem. I have two point clouds which are derived using two RGBD images from ScanNet.
I would like to align both point clouds using the GT poses taken from ScanNet and render an image A using the perspective from B and vice versa.
Therefore, I prepared a colab notebook. 

[https://colab.research.google.com/drive/17h9gevPOHlXzdAZsu1euzk1rUB5bWdA_?usp=sharing](https://colab.research.google.com/drive/17h9gevPOHlXzdAZsu1euzk1rUB5bWdA_?usp=sharing)

I am happy to hear your ideas. Thank you!

**Comment by nikhilaravi** [comment]
Hi there! This is a great question to go into the Discussions channel! I will move it there!
